offchain-modules/packages/x/src/ckb/tx-helper/indexer.ts

```diff
@@ -102,8 +102,8 @@ export class CkbIndexer {
     const infos: IndexerCell[] = [];
     let cursor = null;
     const index = 0;
-    const params = [searchKey, order, `0x${sizeLimit.toString(16)}`, cursor];
     while (true) {
+      const params = [searchKey, order, `0x${sizeLimit.toString(16)}`, cursor];
```

Comment:
L127:

```
 if (liveCells.length < sizeLimit) {
```

should be 

```
if (cursor === '0x')
```

Reply:
ckb indexer 0.2.0 has fixed size_limit problem, besides, using sizeLimit  can call get_cell one time lesser